<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraScheduleWidget</class>
 <widget class="QWidget" name="CameraScheduleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>770</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>924</width>
           <height>756</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,0,0,0,1">
          <property name="topMargin">
           <number>16</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="topButtonsLayout">
            <item alignment="Qt::AlignTop">
             <widget class="nx::vms::client::desktop::CheckBox" name="enableRecordingCheckBox">
              <property name="text">
               <string>Recording</string>
              </property>
              <property name="tristate">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignTop">
             <widget class="QLabel" name="licenseUsageLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string notr="true">&lt;license usage&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item alignment="Qt::AlignTop">
             <widget class="QPushButton" name="licensesButton">
              <property name="text">
               <string>Activate License...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="recordingScheduleLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="scheduleGridGroupBox">
              <property name="title">
               <string notr="true">&lt;Recording Schedule&gt;</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <layout class="QVBoxLayout" name="scheduleGridLayout">
               <item>
                <widget class="nx::vms::client::desktop::ScheduleGridWidget" name="gridWidget" native="true">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>250</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="nx::vms::client::desktop::ScheduleSettingsWidget" name="scheduleSettingsWidget" native="true"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="bottomParametersLayout" stretch="1,1,1">
            <property name="spacing">
             <number>24</number>
            </property>
            <property name="topMargin">
             <number>12</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="nx::vms::client::desktop::ArchiveLengthWidget" name="archiveLengthWidget" native="true"/>
            </item>
            <item>
             <widget class="QGroupBox" name="recordingThresholdGroupBox">
              <property name="title">
               <string>Motion &amp;&amp; Objects Recording</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <layout class="QVBoxLayout" name="recordingThresholdLayout">
               <item>
                <widget class="nx::vms::client::desktop::RecordingThresholdWidget" name="recordingThresholdWidget" native="true"/>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="copyScheduleGroupBox">
              <property name="title">
               <string>Copying Schedule</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1">
               <property name="spacing">
                <number>16</number>
               </property>
               <item>
                <widget class="QPushButton" name="exportScheduleButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>128</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Copy Schedule to...</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::MessageBar" name="hintBar" native="true"/>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::AlertBar" name="recordingAlertBar" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::AlertBar</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/control_bars.h</header>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::ArchiveLengthWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/camera/widgets/archive_length_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::ScheduleGridWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/schedule/schedule_grid_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::ScheduleSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/camera/widgets/schedule_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::RecordingThresholdWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/camera/widgets/recording_threshold_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::MessageBar</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/control_bars.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::CheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/check_box.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>enableRecordingCheckBox</tabstop>
  <tabstop>licensesButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>doNotChangeStateChanged(int)</slot>
 </slots>
</ui>
